setup.py

```diff
@@ -72,7 +72,10 @@ def _get_version() -> str:
             "Operating System :: POSIX :: Linux",
         ],
         python_requires='>=3.8',
-        install_requires=["PyYAML"],
+        install_requires=[
+            'PyYAML',
+            'websocket-client',
```

Comment:
Likewise here, I think pinning the version is probably a good idea.

Reply:
Agreed -- see above.